api/README.md

```diff
@@ -17,6 +17,13 @@ type = 'Kubernetes'
 # This information is crucial if you run PREvant behind a Traefik instance that enforces the user ot be
 # logged in.
 labelsPath = '/run/podinfo/labels'
+
+[runtime.storageConfig]
+# Size of the storage space that is reserved and mounted to the deployed companion with storage.
+storageSize = '10g'
```

Comment:
We should document the default value here.

Reply:
Default is 2g and is mentioned in the document now